Makefile.dep

```diff
@@ -12,6 +12,31 @@ ifneq (,$(filter nhdp,$(USEMODULE)))
   USEMODULE += oonf_rfc5444
 endif
 
+ifneq (,$(filter gnrc_netif_default,$(USEMODULE)))
+  USEMODULE += gnrc_netif
+  USEMODULE += netif_default
```

Comment:
> Maybe not completely, as it might make sense as a pseudo module to pull in netif_default + gnrc_networking, but at least from the board perspective it doesn't make sense to consider.

I meant `gnrc_netif`, not `gnrc_networking`.